### DISCUSSION TITLE
Welcome to mrfioc2 Discussions!

### DISCUSSION THREAD

**Discussion by mdavidsaver** [epics-modules/mrfioc2#77] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zioven** [comment]
Hi,

I am Žiga Oven, a senior software developer at Cosylab, where I am working with EPICS and MRF hardware.

We have several projects where we are deploying MRF timing systems based on MTCA hardware.

We are happy to see the renewed effort to support this module and are happy to contribute back the changes that were made on our projects (a few pull requests were already made by my colleagues).

**Reply by jerzyjamroz** [reply]
All the PRs were addressed and merged. Thanks for the commitment.

**Comment by jerzyjamroz** [comment]
The following releases have been issued: https://github.com/epics-modules/mrfioc2/releases
* 2.3.0 - Community baseline - used for some years already, fixed for any rollback purposes.
* 2.3.1 - Extension of the community baseline with minor changes and patches.
* 2.4.0 - Naming upgrade to the universal standard ```XxxYyy1``` plus ESS, and Cosylab contributions.

**Comment by gabrielfedel** [comment]
Ops, I missed to comment here.
I'm Gabriel Fedel from ESS, and together with Jerzy we work with many MRF hardware (MTCA based).

My initial focus here is to contribute with the documentation, doing some house cleaning first (e.g. moving it from source fourge to github pages),

I hope to be able to help with more improvements on mrfioc2 later.

**Comment by jerzyjamroz** [comment]
@agaget , you are also welcome to share eventual future ideas via the mrfioc2 discussions.

**Comment by agaget** [comment]
Hi,
I am Alexis Gaget,I'm working at CEA Saclay on multiples projects with EPICS and MRF.  
We have deployed or are deploying MRF on  : ESS test stands, Saraf accelerator, IPHI accelerator. 
Mostly MTCA hardware.

With the new efforts on the driver we have been able to finally propose and merge the modifications we have made for SARAF. This accelerator is using the MRF system as timing system but also as a part of its Machine Protection System (MPS) using upstream events and mask in the sequencer. (works well for the MEBT commissionning until now)

I will try to propose some new bob files relying on the old opi files soon.

Cheers

**Reply by gabrielfedel** [reply]
Hi Alexis, we have some nice bob files here at ESS, I will try to make some cleanup and upload them also! But feel free to propose also, no problem!

**Comment by jerzyjamroz** [comment]
@agaget , the release 2.6.1 is out.
